view/frontend/templates/checkout/success.phtml

```diff
@@ -25,7 +25,32 @@
  * @var \Adyen\Payment\Block\Checkout\Success $block
  */
 ?>
-<?php if ($block->isBoletoPayment()): ?>
+
+
+<?php if ($block->renderAction()): ?>
+    <script type="text/javascript">
+        require([
+            'jquery',
+            'adyenCheckout3101'  // the alias for "mage/accordion"
+        ], function ($, AdyenCheckout) {
+            var action = JSON.parse('<?php echo $block->getAction(); ?>');
+            var checkoutComponent = new AdyenCheckout({
+                locale: '<?php echo $block->getLocale(); ?>',
+                environment: '<?php echo $block->getEnvironment(); ?>',
+                originKey: '<?php echo $block->getOriginKey(); ?>'
+            });
+            var actionNode = document.getElementById('ActionContainer');
+            checkoutComponent.createFromAction(action).mount(actionNode);
+        });
+    </script>
+
+
+    <div id="ActionContainer"></div>
+
+
+<?php else: ?>
+
+    <?php if ($block->isBoletoPayment()): ?>
```

Comment:
For phpcs
```suggestion
    </script>
    <div id="ActionContainer"></div>
<?php else: ?>
    <?php if ($block->isBoletoPayment()): ?>
```

Reply:
yes thanks will fix it

Reply:
@acampos1916 this is not correct it need to be in the if statement